doc/src/actioncode.md

```diff
@@ -7,8 +7,11 @@ Action code is normal Rust code with the addition of the following special varia
  * `$1` ... `$n` refer to the respective symbol in the production, numbered
    from 1 (i.e. `$1` refers to the first symbol in the production). If the
    symbol references a rule `R` then an instance of `R`'s type will be stored
-   in the `$i` variable; if the symbol references a lexeme then an
-   `Option<Lexeme>` instance is returned.
+   in the `$i` variable. If the symbol references a lexeme then an
```

Comment:
s/an/a?

Reply:
Fixed in https://github.com/softdevteam/grmtools/pull/197/commits/1ca0379423639a032af3c17e55ef4248b36b1721.